ServiceFabric.PubSubActors/Helpers/SubscriberExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace ServiceFabric.PubSubActors.Helpers
+{
+    /// <summary>
+    /// Enables the below extension method that use the <see cref="SubscribeAttribute"/> to discover message handler methods.
+    /// </summary>
+    public interface ISubscriber
+    {
+    }
+
+    /// <summary>
+    /// Marks a service method as being capable of receiving messages.
+    /// Follows convention that method has signature 'Task MethodName(MessageType message)'
+    /// Polymorphism is supported.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, Inherited = false)]
+    public class SubscribeAttribute : Attribute
+    {
+    }
+
+    public static class SubscriberExtensions
+    {
+        public static Dictionary<Type, Func<object, Task>> DiscoverMessageHandlers(this ISubscriber service)
```

Comment:
This breaks compatibility with the Service-bootstrappers, when removing the `SubscriberServiceHelper`

Reply:
I'd prefer sticking to `ISubscriberService` and `ISubscriberActor` instead of introducing another interface users need to add.

Reply:
Yeah, I'll have to update the bootstrappers.  I introduced `ISubscriber` thinking it would make it possible for users to set up handlers on another class rather than on the Service itself.  I'm not sure how useful that would really be, but that's what I was thinking.  I can take it out and use the existing interfaces only.